lib/travis/yaml.rb

```diff
@@ -22,32 +22,32 @@ module Yaml
     # these are meant as examples. we might want to determine the translation
     # in the client
     MSGS = {
-      alert:           'using a plain string as a secure',
+      alert:           'this string should probably be encrypted',
       alias:           '%{alias} is an alias for %{actual}, using %{actual}',
       cast:            'casting value %{given_value} (%{given_type}) to %{value} (%{type})',
-      default:         'missing %{key}, defaulting to: %{default}',
-      deprecated:      '%{given} is deprecated',
-      downcase:        'downcasing %{value}',
-      edge:            '%{given} is experimental and might be removed in the future',
-      flagged:         'your repository must be feature flagged for %{given} to be used',
-      irrelevant:      'specified %{key}, but this setting is not relevant for the %{on_key} %{on_value}',
+      default:         'missing %{key}, using the default %{default}',
+      deprecated:      '%{given} is deprecated', # Do we need to say stop using it? or?
+      downcase:        'lowercasing %{value}',
```

Comment:
maybe need to reword to this to `using lower case of` or somthing to be more consistent